### PR TITLE
lots of sequelize changes

### PR DESCRIPTION
**Pull request by smaharj1**
- Removed config.json. Those should come from `.env`. So, added that
- Also moved the sequelize sync to the same place as other dbs.
- Added a bunch of MD files (code of conducts, PR and Issue requests)
- Also added `dotenv` for sequelize cause otherwise `process.env` was not detecting any of the `.env` file contents.

### CODE REVIEW

File: bin/express-cli.js [hunk 1]
```diff
@@ -292,9 +293,10 @@ db.sequelize.sync({ force: true }).then(function() {
 `
           mkdir(dir, 'server/models')
           copyTemplateMulti('js/models/sequelize', dir + '/server/models', '*.js')
-          mkdir(dir, 'server/config')
-          copyTemplateMulti('js/models/sequelize/config', dir + '/server/config', '*.json')
-          copyTemplate('js/models/sequelize/config/config.json', path.join(dir, '/server/config/config.json'))
+          // mkdir(dir, 'server/config')
```
Comment (eklemen) [inline bin/express-cli.js]: remove commented out code

File: templates/js/models/sequelize/index.js [hunk 2]
```diff
@@ -1,37 +1,43 @@
...
-  .forEach(function(file) {
-    var model = sequelize["import"](path.join(__dirname, file));
-    db[model.name] = model;
-  });
 
-Object.keys(db).forEach(function(modelName) {
+Object.keys(db).forEach(function (modelName) {
   if (db[modelName].associate) {
-    db[modelName].associate(db);
+    db[modelName].associate(db)
```
Comment (eklemen) [inline templates/js/models/sequelize/index.js]: two different code styles happening. So the cli and generator itself doesnt use semi-colons but the templates that it outputs does...

File: bin/express-cli.js [hunk 3]
```diff
@@ -281,8 +281,9 @@ const db = mongojs(dbUri, collections);
         case 'sequelize':
           pkg.dependencies['mysql2'] = '^1.6.4'
           pkg.dependencies['sequelize'] = '^4.41.2'
+          pkg.dependencies['dotenv'] = '^8.2.0'
           app.locals.localModules.db = './models'
-          www.locals.db = `
+          app.locals.db = `
```
Comment (eklemen) [inline bin/express-cli.js]: need to also remove the logic from `bin/www.js.ejs` thats using www.locals.db

File: bin/express-cli.js (hunk 1 above)
Comment (smaharj1) [reply bin/express-cli.js]: thought you wanted to see it 😆

File: bin/express-cli.js (hunk 3 above)
Comment (smaharj1) [reply bin/express-cli.js]: ah gotcha

**Pull request merged**